drivers/1wire/ds28e17.c

```diff
@@ -45,6 +45,7 @@
 
 #include <nuttx/kmalloc.h>
 #include <nuttx/i2c/i2c_master.h>
```

Comment:
remove too

Reply:
kmalloc include can be remove here but not i2c_master. This existing driver is a 1wire to i2c bridge. So both interfaces are necessary.

Reply:
No can't be removed. Both headers are necessary.